sdk/storage/azure-storage-common/test/test_base.cpp

```diff
@@ -306,4 +246,12 @@ namespace Azure { namespace Storage { namespace Test {
     return secondaryUri.GetAbsoluteUrl();
   }
 
+  bool IsValidTime(const Azure::Core::DateTime& datetime)
+  {
+    // We assume datetime within a week is valid.
```

Comment:
nit: Why? If we really want to assume that, the method should be named to be self-documenting
`IsValidTimeWithinAWeek`

Otherwise, if and when a test fails in the future, and the date is a valid time (well-formed), it will be difficult to debug why by looking at the callstacks or logs, that the reason was it wasn't within one week from now.

Reply:
Nah, I don't think so. The purpose of this function is just to check if a datetime in API response is valid or not, and there really isn't a strict criteria of what should be considered valid. We just assume that none of the test suites will being running for more than a week (which is a safe assumption), so the timestamps for the creation date, last modified date and last accessed date etc won't be older than a week.

If (although it's unlikely to happen) the assumption above is no longer valid in the future, we can totally change the criteria, like extending from a week to a month. Or if we want to adopt a stricter criteria, we can shorten to a day. All we need to do is to change the internal implementation of `IsValidTime`. The callers of this function should not make any changes, because all they want to do is still to check if the timestamp is valid or not.